version/version.go

```diff
@@ -0,0 +1,16 @@
+package version
```

Comment:
Was the new "version" package necessary? Couldn't we have just kept is in one of the existing packages? Since we don't expect that this will grow in functionality.